test/trade/actions/helpers/trade-test.js

```diff
@@ -0,0 +1,248 @@
+import { assert } from 'chai';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+import * as mocks from '../../../mockStore';
+import { augur, abi, constants } from '../../../../src/services/augurjs';
+import { tradeTestState } from '../../constants';
+
+describe('modules/trade/actions/helpers/trade.js', () => {
+	proxyquire.noPreserveCache();
+	const { state, mockStore } = mocks.default;
+	const testState = Object.assign({}, state, tradeTestState);
+	testState.transactionsData = {
+		'trans1': {
+			data: {
+				marketID: '0x000000000000000000000000000000000binary1',
+				outcomeID: '2',
+				marketType: 'binary',
+				marketDescription: 'test binary market',
+				outcomeName: 'YES'
+			},
+			feePercent: {
+				value: '0.199203187250996016'
+			}
+		},
+		'trans2': {
+			data: {
+				marketID: '0x0000000000000000000000000000categorical1',
+				outcomeID: '1',
+				marketType: 'categorical',
+				marketDescription: 'test categorical market',
+				outcomeName: 'Democratic'
+			},
+			feePercent: {
+				value: '0.099800399201596707'
+			}
+		},
+		'trans3': {
+			data: {
+				marketID: '0x000000000000000000000000000000000scalar1',
+				outcomeID: '1',
+				marketType: 'scalar',
+				marketDescription: 'test scalar market',
+				outcomeName: ''
+			},
+			feePercent: {
+				value: '0.95763203714451532'
+			}
+		}
+	};
+	testState.orderBooks = {
+		'0x000000000000000000000000000000000binary1': {
```

Comment:
I'm going to change these to human readable skins, something like `testBinaryMarketID` so that it's very clear what type of market we are looking to test against. I would be happy to change to hex strings if it is prefered but I like the human readables in this case just to make it clearer when looking at the tests.